lib/mixlib/install/backend/package_router.rb

```diff
@@ -236,6 +235,8 @@ def omnibus_project
         def normalize_architecture(architecture)
           if %w{ sun4u sun4v }.include?(architecture)
             architecture = "sparc"
+          elsif %w{ i686 }.include?(architecture)
+            architecture = "i386"
           end
```

Comment:
We need to expand this mapping:

```
if %w{ x86_64 amd64 x64 }.include?(architecture)
  "x86_64"
elsif %w{ i386 x86 i86pc i686 }.include?(architecture)
  "i386"
elsif %w{ sparc sun4u sun4v }.include?(architecture)
  "sparc"
elsif %w{ ppc64le ppc64el }.include?(architecture)
  "ppc64le"
```

I pulled most that logic from:
https://github.com/chef/mixlib-install/blob/v1.2.3/lib/mixlib/install/backend/bintray.rb#L207-L279

I would look that file over and see if you think any other mappings/normalization needs to be done. Please note that the old method was more complicated as we had to infer the architecture from the filename (vs now we have a nice `omnibus.architecture` Artifactory property which contains the architecture value as computed by Omnibus).